soc/riscv/nordic_nrf/common/vpr/Kconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config RISCV_CORE_NORDIC_VPR
```

Comment:
note for future: we should think of moving core-dependent code to arch/riscv/cores, or soc/common since we could potentially have independent socs running the same riscv cores. not likely the case for VPR though.